pyppl/console/plugins.py

```diff
@@ -0,0 +1,29 @@
+"""List all available plugins, including runners, even they are temporarily disabled."""
+from ..plugin import hookimpl, pluginmgr
+from ..runner import runnermgr, _runner_name
+from ..logger import logger
+
+@hookimpl
+def cli_addcmd(commands):
+	"""Add plugins command"""
+	commands.plugins._hbald = False
+	commands.plugins        = __doc__
+
+@hookimpl
+def cli_execcmd(command, opts): # pylint: disable=unused-argument
+	"""Run the command"""
+	if command == 'plugins':
+		logger.init({'level': 'notset'})
+		for plugin in sorted(pluginmgr.get_plugins(),
+			key = lambda plug: pluginmgr.get_name(plug)):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Lambda may not be necessary](https://app.codacy.com/app/pwwang/PyPPL/pullRequest?prid=4727008)